modules/distribution/src/assembly/bin.xml

```diff
@@ -380,5 +385,10 @@
             <destName>config.json</destName>
             <fileMode>644</fileMode>
         </file>
+        <file>
```

Comment:
Aren't we supposed to add database and the widgets through a feature?

Reply:
I have copied my database here, we do not need to write the feature for database and the widgets.

Reply:
Would n't this be overwritten when dashboard feature installs the same database over this?